src/connectedk8s/azext_connectedk8s/_validators.py

```diff
@@ -4,6 +4,10 @@
 # --------------------------------------------------------------------------------------------
 
 
+from os import name
```

Comment:
pls validate that these modules are installed with cli core 2.16

Reply:
ack

Reply:
i tested & it works with 2.16